fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/tree/lowcardinality/DecodeContext.java

```diff
@@ -193,7 +202,118 @@ private ColumnRefOperator createNewDictColumn(ColumnRefOperator column) {
         }
     }
 
-    private class FunctionRewriter extends BaseScalarOperatorShuttle {
+    // define mode: means the result column is dict, DictExpr should return int/array<int> type
```

Comment:
encode and decode mode? not define mode.

Reply:
I want to express is that this column is a new dict-column and it's defined by the expression, it's not encode string column to dict column